hudi-hadoop-common/src/main/java/org/apache/hudi/io/storage/HoodieHadoopAvroParquetReader.java

```diff
@@ -52,14 +52,14 @@
 /**
  * {@link HoodieFileReader} implementation for parquet format.
  */
-public class HoodieAvroParquetReader extends HoodieAvroFileReaderBase {
+public class HoodieHadoopAvroParquetReader extends HoodieAvroFileReaderBase {
```

Comment:
I think we can keep the existing reader and writer class names.  Just rename the factory class names to say it's Hadoop-based.